Makefile

```diff
@@ -68,3 +68,5 @@ docker-build:
 
 docker-run:
 	@npm run docker
+
+.PHONY: build test
```

Comment:
Oh nice catch. I suppose `clean` would have let `build` run again, but poor `test`. Should we add the rest of the commands to PHONY in case of name collisions. I don't believe we actually have any target files we want to make here.

Hah, and I only now just noticed that @blowery added a typo `urn` ⚱ command.